agent/containers/images/image-test-script.sh

```diff
@@ -133,6 +200,9 @@ printf -- "\n\nTypically pbench-tool-meister-start is expecting a '\${benchmark_
 
 wait_keypress 120
 
+printf -- "\n\nNow we will begin with data collection, but before doing so, feel free to run this on any tool-meister hosts:\n\n\t$ podman run docker.io/alexeiled/stress-ng --cpu 4 --io 2 --timeout 300s --metrics-brief\n\nThis will serve as a sample workload to demonstrate meaningful data collection.\nIt runs for 300s by default, but feel free to alter that value (or ctrl-c for early graceful exit).\n\n"
```

Comment:
Um...could we possibly break this up into readable lines?  There are several nice ways to do that...here's one:
```
cat <<-__EOF__


Now we will begin with data collection, but before doing so, feel free to run this on any tool-meister hosts:

    $ podman run docker.io/alexeiled/stress-ng --cpu 4 --io 2 --timeout 300s --metrics-brief

This will serve as a sample workload to demonstrate meaningful data collection.
It runs for 300s by default, but feel free to alter that value (or ctrl-c for early graceful exit).

__EOF__
```